app/models/ncr/work_order.rb

```diff
@@ -26,6 +26,7 @@ def self.purchase_amount_column_name
     before_validation :normalize_values
     before_update :record_changes
 
+    validates :amount, presence: true
```

Comment:
spec for this? Looks like we don't have any validation specs in the spec file for this model -- would love to hear about why! Is there a philosophical opposition to specs for AR validations (I know many hold the view that they are not important because it's basically just double checking that you wrote a validation rather than testing behavior)